src/samples/server/Elsa.Samples.Server.Host/Startup.cs

```diff
@@ -87,9 +88,13 @@ public void ConfigureServices(IServiceCollection services)
                 .AddElsaApiEndpoints()
                 .AddElsaSwagger();
 
+            services.AddJavaScriptTypeDefinitionProvider<CustomerTypeDefinitionProvider>();
+            services.AddJavaScriptTypeDefinitionProvider<ClientTypeDefinitionProvider>();
+
             // Allow arbitrary client browser apps to access the API for demo purposes only.
             // In a production environment, make sure to allow only origins you trust.
-            services.AddCors(cors => cors.AddDefaultPolicy(policy => policy.AllowAnyHeader().AllowAnyMethod().AllowAnyOrigin().WithExposedHeaders("Content-Disposition")));
+            //services.AddCors(cors => cors.AddDefaultPolicy(policy => policy.AllowAnyHeader().AllowAnyMethod().AllowAnyOrigin().WithExposedHeaders("Content-Disposition")));
+            services.AddCors(cors => cors.AddPolicy("ClientPermission", policy => policy.AllowAnyHeader().AllowAnyMethod().WithOrigins("http://localhost:3333").AllowCredentials().WithExposedHeaders("Content-Disposition")));
```

Comment:
Can we change this back to allow **any** origin?

Reply:
Fixed